javax/ws/rs/core/MediaType.hpp

```diff
@@ -0,0 +1,156 @@
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#ifndef NATIVE_MEDIATYPE_H
+#define NATIVE_MEDIATYPE_H
+
+#include "../../../../java/Lang.hpp"
+#include "../../../../java/util/HashMap/HashMap.hpp"
+
+namespace Javax {
+    namespace Ws {
+        class MediaType : public virtual Java::Lang::Object {
+        public:
+            static String APPLICATION_ATOM_XML;
+            static String APPLICATION_FORM_URLENCODED;
+            static String APPLICATION_JSON;
+            static String APPLICATION_OCTET_STREAM;
+            static String APPLICATION_SVG_XML;
+            static String APPLICATION_XHTML_XML;
+            static String APPLICATION_XML;
+            static String MEDIA_TYPE_WILDCARD;
+            static String MULTIPART_FORM_DATA;
+            static String TEXT_HTML;
+            static String TEXT_PLAIN;
+            static String TEXT_XML;
+            static String WILDCARD;
+            static MediaType APPLICATION_ATOM_XML_TYPE;
+            static MediaType APPLICATION_FORM_URLENCODED_TYPE;
+            static MediaType APPLICATION_JSON_TYPE;
+            static MediaType APPLICATION_OCTET_STREAM_TYPE;
+            static MediaType APPLICATION_SVG_XML_TYPE;
+            static MediaType APPLICATION_XHTML_XML_TYPE;
+            static MediaType APPLICATION_XML_TYPE;
+            static MediaType MULTIPART_FORM_DATA_TYPE;
+            static MediaType TEXT_HTML_TYPE;
+            static MediaType TEXT_PLAIN_TYPE;
+            static MediaType TEXT_XML_TYPE;
+            static MediaType WILDCARD_TYPE;
+
+            /**
+             * Creates a new instance of MediaType, both type and subtype are wildcards.
+             */
+            MediaType();
+
+            /**
+             * Creates a new instance of MediaType with the supplied type and subtype.
+             *
+             * @param type
+             * @param subtype
+             */
+            MediaType(const String &type, const String &subtype);
+
+            /**
+             *   Creates a new instance of MediaType with the supplied type, subtype and parameters.
+             *
+             * @param type
+             * @param subtype
+             * @param parameters
+             */
+            MediaType(const String &type, const String &subtype, const HashMap<String, String> &parameters);
+
+            /**
+             * Remove media type instance
+             */
+            ~MediaType();
+
+            /**
+             *  Getter for a read-only parameter map.
+             *
+             * @return
```

Comment:
I follow this documentation, How to write doc comment for Java Doc Tool
Please show me the rule Data type after @String and I will fix it